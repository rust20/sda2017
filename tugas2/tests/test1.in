11 5 5 50 7 4
Arturia Pendragon;50;100;pedang
Amakusha;40;99;pedang
Sishou;1;99;panah
Atlanta;1;10;panah
Emiye;25;10;panah
Jyanu;22;100;pedang
Hector;1;11;panah
Hercules;51;10000;pedang
Gilgamesh;1;9999;panah
Cu;1;100;panah
Kizuna;1000;1000;pedang
5;2;Gilgamesh,Hercules
3;2;Arturia Pendragon,Amakusha,Sishou
1;3;Atlanta,Emiye,Jyanu,Hector
7;2;Cu
7;1;Kizuna
4;2;1;1;panah;3
2;2;25;1;panah;3
2;1;40;3;pedang;4
4;4;201;30;pedang;1
1;1;10000;2;pedang;1000
D.S#
DD##
#S##
#D#D
#S#.
#M#.
SS..
