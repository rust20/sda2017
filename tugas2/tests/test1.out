5,2 tidak ada pahlawan yang ikut
4,2 RUN, kekuatan maksimal sejumlah: 0
3,2 Pahlawan yang ikut:Arturia Pendragon
2,2 BATTLE, kekuatan: 50, pahlawan: Arturia Pendragon
1,3 Pahlawan yang ikut:Jyanu,Hector,Atlanta,Emiye
1,1 RUN, kekuatan maksimal sejumlah: 262
2,1 BATTLE, kekuatan: 242, pahlawan: Arturia Pendragon,Jyanu,Hector,Atlanta
7,2 Pahlawan yang ikut:Cu
4,4 RUN, kekuatan maksimal sejumlah: 200
7,1 tidak ada pahlawan yang ikut
Akhir petualangan Gudako
Level Gudako: 14
Level pahlawan:
Arturia Pendragon: 5
Jyanu: 4
Hector: 4
Atlanta: 4
Emiye: 1
Cu: 1
